packages/vite/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@vocab/vite",
+  "version": "1.0.0",
```

Comment:
Not that we're going to publish this right now, but would prefer the package start at `0.0.1` to signify that this plugin doesn't have a stable API yet.

Reply:
Cool, What's the usual convention when starting packages?

This is v0.u0.p1?

Reply:
I guess it depends whether we're confident that we won't want to make breaking changes any time soon. Given that this is the first iteration of the vite plugin, and that we haven't tested it in a real application yet, I'd rather have the flexibility to make breaking changes without bumping the major version.